app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -182,6 +194,7 @@ export default {
     };
   },
   computed: {
+    ...mapGetters('auth', ['tokenParsed', 'user']),
```

Comment:
No sure if we need stuff in the Form Designer for the token? Could be wrong, worth a discussion.

Like is a token needed at design time at all? That would be the logged in user _creating_ the form at that time...

Reply:
Figured it out, this lets you see the result of what you set in Custom Default Value at design time 👍